src/go/rpk/pkg/cli/cmd/acl/list.go

```diff
@@ -99,63 +103,102 @@ func describeReqResp(
 	printAllFilters bool,
 	printMatchesHeader bool,
 	b *kadm.ACLBuilder,
+	format string,
 ) {
 	results, err := adm.DescribeACLs(context.Background(), b)
 	out.MaybeDie(err, "unable to list ACLs: %v", err)
 	types.Sort(results)
 
-	// If any filters failed, or if all filters are
-	// requested, we print the filter section.
-	var printFailedFilters bool
-	for _, f := range results {
-		if f.Err != nil {
-			printFailedFilters = true
-			break
+	filteredAndDescribedResults := filteredAndDescribedResults{}
```

Comment:
This is the first time I've pushed a PR to an open source project, I didn't even think to look for contributing document, silly oversight on my part, my apologies. How would you like me to proceed on future updates then? I thought about the one PR per command method at the start, but I wasn't sure when dev would go into a release and did wasn't sure if it would be okay for a new version of rpk to be launched with only partial `--format` support.  

If that's method is preferred to make reviews easier. I'll close out this PR and open just one (topic list) as mentioned tonight to reduce review scope, then open up new PRs for each as they're accepted/merged.